src/test/java/io/jenkins/plugins/analysis/core/scm/NullBlamerTest.java

```diff
@@ -0,0 +1,37 @@
+package io.jenkins.plugins.analysis.core.scm;
+
+import org.junit.jupiter.api.Test;
+
+import edu.hm.hafner.analysis.Report;
+
+import static io.jenkins.plugins.analysis.core.assertions.Assertions.*;
+import static org.mockito.Mockito.*;
+
+/**
+ * Tests the class {@link NullBlamer}
+ *
+ * @author Florian Hageneder
+ */
+class NullBlamerTest {
+
+    @Test
+    void shouldLogSkippedBlaming() {
+        Report report = mock(Report.class);
+
+        final NullBlamer blamer = new NullBlamer();
```

Comment:
No `final` for local variables, see [coding style](https://github.com/uhafner/codingstyle).